### PR TITLE
[Merged by Bors] - chore: golf `FunctionsBoundedAtInfty`

### PR DESCRIPTION
**Pull request by urkud**
Use lemmas about `Filter.HasBasis` to golf some proofs.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/Complex/UpperHalfPlane/FunctionsBoundedAtInfty.lean [hunk 1]
```diff
@@ -85,26 +78,9 @@ theorem bounded_mem (f : ℍ → ℂ) :
...
-    obtain ⟨A, hA⟩ := h (ε / 2) h11
-    use A
-    intro z hz
-    have hzs : f z ∈ s := by
-      apply h2
-      simp only [mem_ball_zero_iff, norm_eq_abs]
-      apply lt_of_le_of_lt (hA z hz)
-      linarith
-    apply hzs
+    IsZeroAtImInfty f ↔ ∀ ε : ℝ, 0 < ε → ∃ A : ℝ, ∀ z : ℍ, A ≤ im z → abs (f z) ≤ ε :=
```
Comment (ocfnash) [inline Mathlib/Analysis/Complex/UpperHalfPlane/FunctionsBoundedAtInfty.lean:81]: How about:
```suggestion
    IsZeroAtImInfty f ↔ ∀ ε > 0, ∃ A : ℝ, ∀ z : ℍ, A ≤ im z → abs (f z) ≤ ε :=
```

**Review by ocfnash — APPROVED** [review]
Thanks!

bors d+

File: Mathlib/Analysis/Complex/UpperHalfPlane/FunctionsBoundedAtInfty.lean (hunk 1 above)
Comment (urkud) [reply Mathlib/Analysis/Complex/UpperHalfPlane/FunctionsBoundedAtInfty.lean:81]: I would prefer not to change the statement now (even up to a reducible definition).

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**